release-schema.json

```
@@ -14,7 +14,45 @@
         "legalBasis": {
           "title": "Legal basis",
           "description": "The legal basis of the procedure.",
-          "$ref": "#/definitions/Classification"
+          "type": "object",
+          "properties": {
+            "scheme": {
+              "title": "Scheme",
+              "description": "The scheme or codelist from which the legislation's code is taken.",
```

I think 'code' makes sense in the context of a classification, but 'identifier' makes more sense in this context - like in the `Identifier` object.

```suggestion
              "description": "The identifier scheme from which the legislation's identifier is taken.",
```